pebblo/app/service/prompt_service.py

```diff
@@ -196,6 +197,9 @@ def process_request(self):
         except ValidationError as ex:
             logger.error(f"Error in Prompt API process_request. Error:{ex}")
             raise HTTPException(status_code=400, detail=str(ex))
+        except FieldValidationException as ex:
```

Comment:
Since the `except` block is the same as above, you can simply combine it instead of duplicating it.
`except (ValidationError, FieldValidationException) as ex`